Makefile

```diff
@@ -1,4 +1,10 @@
-REGISTRY?=registry.devshift.net
+ifeq ($(TARGET),rhel)
+  DOCKERFILE := Dockerfile.rhel
+  REGISTRY := push.registry.devshift.net/osio-prod
+else
+  DOCKERFILE := Dockerfile
+  REGISTRY := registry.devshift.net
```

Comment:
registry should always be `push.registry.devshift.net` I think

Reply:
fixed, thanks